### DISCUSSION TITLE
404 page error with dev.json request

### DISCUSSION THREAD

**Discussion by pixelpaulaus** [nuxt/nuxt#27624] [Questions]
HI all,
My Nuxt static site build seems to work without error. But then, when I visit a specific page, a request for `/_nuxt/builds/meta/dev.json` is made, and then the whole page disappears and shows the error page.
But it only happens on some builds. So if I rebuild it again, it may not happen. But it is impossible to know if it will without manually testing every single page after the build, which is not good.
Would anyone know why this randomly happens and how to stop it?
I am using the latest nuxt.

**Comment by kigathi-chege** [comment]
I am experiencing the same error.
Nuxt 3.10.2
macOS Sonoma 14.5

**Reply by kyejune** [reply]
also 3.13.1

[a comment by an automated account is not shown]

**Comment by Hautzii** [comment]
You can put `experimental: { appManifest: false },` in your nuxt.config.ts file as a workaround.

**Reply by amery** [reply]
Getting this with `3.16.0`. work around still valid.

**Reply by charanpreet-indee** [reply]
Issue still exist on `3.17.0` with workaround still working.

**Reply by educbraga** [reply]
Issue still exist on 3.17.6

**Comment by OrbisK** [comment]
Can someone share a stackblitz reproduction?
